### PR TITLE
chore: add more Web3 wallets

### PR DESCRIPTION
**Pull request by eragon512**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**
Manual

Screenshot 2024-02-17 at 12 56 35 AM

### CODE REVIEW

File: template/web/src/store/createWagmiConfig.ts [hunk 1]
```diff
@@ -13,7 +21,14 @@ export function createWagmiConfig(projectId: string) {
       },
       {
         groupName: 'Other Wallets',
-        wallets: [rainbowWallet, metaMaskWallet],
+        wallets: [
+          rainbowWallet,
+          metaMaskWallet,
+          trustWallet,
```
Comment (Sneh1999) [inline template/web/src/store/createWagmiConfig.ts:27]: @eragon512 have you tested with these wallets?

File: template/web/src/store/createWagmiConfig.ts [hunk 2]
```diff
@@ -13,7 +21,14 @@ export function createWagmiConfig(projectId: string) {
...
+          braveWallet,
+          ledgerWallet,
+          walletConnectWallet,
```
Comment (Zizzamia) [inline template/web/src/store/createWagmiConfig.ts:30]: Wallet Connect adds trash in the HTML, let's not add it. We can change our mind after they clean up their library.

File: template/web/src/store/createWagmiConfig.ts (hunk 1 above)
Comment (Zizzamia) [reply template/web/src/store/createWagmiConfig.ts:27]: And for each Wallet, I would like we triple check the HTML is trashed.

**Comment by eragon512** [comment]
Not able to test the other wallets at all, closing this PR for now

**Pull request closed without merging**